### ISSUE TITLE
Walkthrough Page on Tidy Data/Dplyr

### ISSUE THREAD

**Issue by zachbogart** [jtr13/EDAV#140]
Think it would be helpful to walkthrough a simple dataset and how to manipulate it/make it tidy using tidyverse packages

- Should take dataset from untidy to tidy with a bunch of lovely explanation along the way.
- Could like to DataCamp courses relevant to the subject
- Maybe end with a simple graph of the resulting dataframe

**Comment by jtr13** [comment]
I have a tutorial on tidyr::gather() which covers a lot of this:

https://github.com/jtr13/codehelp/blob/master/R/gather.md

I have some local edits which I need to push so check the date first.

Dplyr could be another section.

**Comment by akshatapatel** [comment]
Hi, i would like to contribute to this issue for my community contributing. Let me know how to proceed with it.

**Comment by zachbogart** [comment]
Good to hear! I would start by thinking about how you want structure the chapter. Please let us know what you are thinking and we can push you in the right direction.

**Comment by zachbogart** [comment]
I think for a walkthrough page, you should probably steer towards going from messy to a tidy dataset. Along the way, try to assume the person you are writing it for in new to dplyr. You could explain what you are doing with the main verbs, link to hadley's stuff on tidy data, etc. 

There is only one other walkthrough page for reference, but I tried to make it so that someone could follow along and understand every step. This is different from the green pages where it is basically just by example and students can discover it for themselves.

**Comment by akshatapatel** [comment]
Can the airquality data in the datasets package be considered as messy?
<img width="440" alt="screen shot 2018-10-28 at 3 29 27 pm" src="https://user-images.githubusercontent.com/2765148/47621373-2218c680-dacd-11e8-9966-dceca03f1f5f.png">

**Comment by akshatapatel** [comment]
I am trying to make a walkthrough to tidy a messy data

**Comment by zachbogart** [comment]
I see. So this is a dataset you are using to create the walkthrough. I would look for a dataset that has a more blatant example of untidyness. For inspiration:
https://ramnathv.github.io/pycon2014-r/explore/tidy.html

**Comment by jtr13** [comment]
Agree, a good clue that the dataset is already tidy is that the units are different for each column.
